eos-tech-support/eos-procmem-sample

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+declare -A rss_sums
+declare -A pss_sums
+
+if [ $EUID != 0 ]; then
+    echo "permission denied: must be run as root" >&2
+    exit 1
+fi
+
+for pid in $(ps -ef | awk '{print $2}'); do
+    if [ -f /proc/$pid/smaps ]; then
+    cmd=$(cat /proc/$pid/comm)
```

Comment:
Fix indentation for this entire `then` paragraph

Reply:
Fixed in the new version.